tla/consistency/ExternalHistoryInvars.tla

```diff
@@ -205,7 +205,56 @@ CommittedRwLinearizableInv ==
     /\ AllCommittedObservedInv
     /\ AtMostOnceObservedInv
 
-\*Debugging invariants to check that specific states are reachable
+----
+
+\* Ordering over txIDs, the form of which is <<view,seqnum>>
+TxIDStrictlyLessThan(x, y) ==
+    \/ x[1] < y[1]
+    \/ /\ x[1] = y[1]
+       /\ x[2] < y[2]
+
+\* CommittedRwResponseSorted is a subset of history containing only the responses to committed rx transactions
```

Comment:
Dangling "Sorted" in "CommittedRwResponseSorted"?

Reply:
Fixed in https://github.com/microsoft/CCF/pull/6185/commits/fa696c6be749aa98819432572d2ee53cd4b4b157